### PR TITLE
Menu burger

### PR DESCRIPTION
**Pull request by ghost**
Add the header, the logo and the menu burger, we just need to stylish that ;)

### CODE REVIEW

**Review by Dash293 — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

**Review by LoicBrassart — DISMISSED** [review]
Saw no Evil

**Review by Dash293 — DISMISSED** [review]

File: frontend/src/components/Header/style.jsx [hunk 1]
```diff
@@ -1,5 +1,147 @@
...
+    min-height: 75px;
+  }
+  .navbarsolid {
+    background-color: green;
+    color: yellow;
+  }
+  .navbar_logo {
+    justify-self: center;
+  }
+  .navbar_links {
```
Comment (LuigiMenez) [inline frontend/src/components/Header/style.jsx]: First time

Comment (LuigiMenez) [inline frontend/src/components/Header/style.jsx]: First time

File: frontend/src/components/Header/style.jsx [hunk 2]
```diff
@@ -1,5 +1,147 @@
...
+    display: flex;
+  }
+  .navbar_link {
```
Comment (LuigiMenez) [inline frontend/src/components/Header/style.jsx]: second time

File: frontend/src/components/Header/style.jsx [hunk 3]
```diff
@@ -1,5 +1,147 @@
...
+    color: inherit;
+  }
+  .navbar_burger {
+    width: 40px;
+    height: 40px;
+    background: transparent;
+    border: none;
+    color: inherit;
+  }
+  .navbar_links {
```
Comment (LuigiMenez) [inline frontend/src/components/Header/style.jsx]: second time

File: frontend/src/components/Header/style.jsx (hunk 2 above)
Comment (LuigiMenez) [inline frontend/src/components/Header/style.jsx]: First time

Comment (LuigiMenez) [inline frontend/src/components/Header/style.jsx]: First time

File: frontend/src/components/Header/style.jsx [hunk 4]
```diff
@@ -1,5 +1,147 @@
...
+  }
+  .navbar_item::after {
+    content: "";
+    display: block;
+    margin: 0 auto;
+    width: 3vw;
+    height: 1px;
+    background: #fff;
+  }
+  .navbar_link {
```
Comment (LuigiMenez) [inline frontend/src/components/Header/style.jsx]: second time

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
It is better to use only one writing standard for the class name, preferred the camelCase

**Review by Dash293 — DISMISSED** [review]

**Review by Dash293 — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**